src/styles/components/_form.scss

```diff
@@ -142,6 +142,8 @@ select:not([multiple]) {
 
   &:focus,
   &.focus {
+    border-color: $dso-anchor-focus-color;
+    border-width: 3px;
```

Comment:
Waarom gebruik je de ene keer `border-width` en de andere keer `outline`?

En daarbij, het element zonder `:focus` heeft geen border van 3px. Stuitert het element nu niet als het focus krijgt?

Reply:
Wij vinden het niet stuiteren. Dat border-width wordt gebruikt is omdat dan alleen de border van een form moet worden aangepast anders moet border en outline aangepast worden. Om een mengelmoes van 2 kleuren kort op elkaar te krijgen.

Reply:
Je hebt gelijk, ik had niet scherm dat dat `border-box` zijn, en geen `content-box`.